app/views/groups/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<% if current_user.groups.none? %>
+  <div class="introduction">
+    <h2>Welcome to Rails-Wallet App! <%= current_user.name %></h2>
+    <p>With this app, you can easily keep track of your expenses and income by adding transactions. You can group your transactions together to get a better overview of your spending and earnings This app is designed to be user-friendly and easy to navigate, so you can quickly and easily manage your finances. Whether you're trying to save money, pay off debt, or just want to keep better track of your spending, this app can help you achieve your financial goals. Start using this app today and take control of your money!</p>
+  </div>
+<% end %>
+
+  <section class="groups-container">
+    <% @groups.each do |group| %>
+        <%= link_to user_group_path(current_user, group) do %>
+          <div class="group-card">
+              <% if group.icon.attached? %>
+                <%= image_tag group.icon, class: "group-icon" %>
+              <% else %>
+                <%= image_tag "default_product_image.webp", class: "group-icon" %>
+              <% end %>
+              <h3> <%= group.name %> </h3>
+              <p id="total-amount">$<%= group.entities.sum(:amount) %></p>
+          </div>
+        <% end %>
+    <% end %>
+    <%= link_to "Add New Category", new_user_group_path(current_user), class: "add-btn green-btn" %>
+  </section>
```

Comment:
- [ ] As requested by the previous reviewer,  Kindly you need to improve your design, Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
- [ ] Kindly follow the provided design (width, font, size, height, background, color...)
1. The `ADD NEW TRANSACTION` button must always be at the bottom of your page. ⬇️  
2. Add this white border around the ADD NEW CATEGORY button text. 👍🏻
3. The Add Transaction button has a similar problem. Kindly fix it as well. 💯  👍🏻

- Original
![image](https://user-images.githubusercontent.com/79058364/230731290-6f44f560-b369-4d7e-9e4b-51130a17508b.png)

- Your Button
![image](https://user-images.githubusercontent.com/79058364/236656984-514f0a5c-c392-4a81-934d-eceeee661892.png)


![image](https://user-images.githubusercontent.com/79058364/236657236-6311f5a3-5ae4-4b8a-820c-6251c1f1027b.png)